src/AWSS3.jl

```diff
@@ -124,7 +123,14 @@ function s3_get(
             params["headers"] = headers
         end
 
-        return S3.get_object(bucket, path, params; aws_config=aws, kwargs...)
+        r = S3.get_object(bucket, path, params; aws_config=aws, kwargs...)
+        return if return_stream
+            r.io
```

Comment:
Will probably break things as this now returns a `IOBuffer` instead of the `BufferStream`. What we could do is pass in `"response_stream" => Base.BufferStream()` and then close it before returning it.

Reply:
Where does that argument get passed to?